config/module/get_test.go

```diff
@@ -26,6 +25,10 @@ type testMod struct {
 	version  string
 }
 
+const (
+	testCredentials = "a9564ebc3289b7a14551baf8ad5ec60a"
```

Comment:
What is this token? Is it just a fake thing, or is it something real on some registry?

If it's something fake then it could be clearer to use something more obviously-fake, like `"fake-token"`. Or at least have a common explaining that for the benefit of future maintainers wondering what this number means. :grinning:

Reply:
yes, it's just random. You're right, probably should make this more obvious 👍